app/models/miq_worker.rb

```diff
@@ -383,39 +341,25 @@ def systemd_worker?
   end
 
   def start_runner
-    if ENV['MIQ_SPAWN_WORKERS'] || !Process.respond_to?(:fork)
-      start_runner_via_spawn
-    elsif systemd_worker?
+    if ENV['MIQ_SYSTEMD_WORKERS'] && systemd_worker?
       start_systemd_worker
     elsif containerized_worker?
       start_runner_via_container
     else
-      start_runner_via_fork
+      start_runner_via_spawn
     end
   end
 
   def start_runner_via_container
     create_container_objects
   end
 
-  def start_runner_via_fork
-    self.class.before_fork
-    pid = fork(:cow_friendly => true) do
-      self.class.after_fork
-      self.class::Runner.start_worker(worker_options)
-      exit!
-    end
-
-    Process.detach(pid)
-    pid
-  end
-
   def self.build_command_line(guid, ems_id = nil)
     raise ArgumentError, "No guid provided" unless guid
 
     require 'awesome_spawn'
     cmd = "#{Gem.ruby} #{runner_script}"
-    cmd = "nice #{nice_increment} #{cmd}" if ENV["APPLIANCE"]
+    cmd = "nice -n #{nice_increment} #{cmd}" if ENV["APPLIANCE"]
```

Comment:
I'm not sure when nice went to `-n` format but I think it's been a long time, since we went to fork, so this is why this is changed here.